src/main.rs

```diff
@@ -280,35 +243,56 @@ impl Inlyne {
                             if !scrollbar_held {
                                 scrollbar_held = true;
                             }
-                        } else if click_scheduled && !jumped {
+                        } else if self.renderer.selection.is_none() {
```

Comment:
Although updating tuple values is very cheap, I think it makes more sense and is more efficient if selection is only set on a click pressed event. To prevent the issue of selection when jumping 

```
+     } else if self.renderer.selection.is_none() {
+         // Only set selection when not over link
+         self.renderer.selection = Some((last_loc, last_loc));
        }
+ } else if self.renderer.selection.is_none() {
+     self.renderer.selection = Some((last_loc, last_loc));
+ }
``` 
can be added [here](https://github.com/trimental/inlyne/pull/23/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR284)